spec/duration.html

```diff
@@ -710,17 +845,17 @@ <h1>ToTemporalDurationRecord ( _temporalDurationLike_ )</h1>
               [[Microseconds]]: _temporalDurationLike_.[[Microseconds]],
               [[Nanoseconds]]: _temporalDurationLike_.[[Nanoseconds]]
             }.
-        1. Let _result_ be a new Record with all the internal slots given in the Internal Slot column in <emu-xref href="#table-temporal-temporaldurationlike-properties"></emu-xref>.
+        1. Let _result_ be a new Duration Record.
         1. Let _any_ be *false*.
-        1. For each row of <emu-xref href="#table-temporal-temporaldurationlike-properties"></emu-xref>, except the header row, in table order, do
-          1. Let _prop_ be the Property value of the current row.
+        1. For each row of <emu-xref href="#table-temporal-duration-record-fields"></emu-xref>, except the header row, in table order, do
+          1. Let _prop_ be the Property Name value of the current row.
           1. Let _val_ be ? Get(_temporalDurationLike_, _prop_).
           1. If _val_ is *undefined*, then
-            1. Set _result_'s internal slot whose name is the Internal Slot value of the current row to 0.
+            1. Set _result_'s field whose name is the Field Name value of the current row to 0.
```

Comment:
I do think it's out of scope for this PR... I'm trying not to fix any inconsistent types (except for the obvious ones in 1329379efa3b325888c4b31e5ee29d069f3d2b2e) but instead do all the necessary refactoring in this PR so that the follow-up that actually fixes the types is clearer and easier to review.
(BTW, in my draft locally I'm actually removing the 𝔽() a couple of lines down, instead of changing this to 0_𝔽)